src/lib/components/get-demo/calendly-embed.svelte

```diff
@@ -0,0 +1,20 @@
+<style lang="postcss">
+  div {
+    @media (min-width: 1091px) {
+      @apply -mt-small;
+    }
+  }
+</style>
+
+<svelte:head>
+  <script
+    type="text/javascript"
+    src="https://assets.calendly.com/assets/external/widget.js"
+    async></script>
```

Comment:
Remove `async`. (Ref [this](https://help.calendly.com/hc/en-us/articles/360019861794-Common-embed-questions#1))

```suggestion
    src="https://assets.calendly.com/assets/external/widget.js"></script>
```